demo/config/database.zig

```diff
@@ -4,7 +4,7 @@ pub const database = .{
         .adapter = .null,
     },
     .testing = .{
-        .adapter = .null,
+        .adapter = .postgresql,
```

Comment:
OK. There's one issue with this. This makes it so any new project, when they do `jetzig init`, this setting gets populated in their new project.

This is a _new_ behavior and I want to call this out.

Reply:
OK. I found the correct location. https://github.com/jetzig-framework/jetzig/blob/main/cli/commands/init.zig#L109

Perhaps we should create a `database_init.zig` file and leave that for use by the CLI tool for init?

Reply:
Done